### ISSUE TITLE
feat: Add an option to enter API URL manually for CCSync

### ISSUE THREAD

**Issue by its-me-abhishek** [CCExtractor/taskwarrior-flutter#392]
### Describe your improvement idea or feature request

An option should be there to input the API url directly, without recompiling the app.

### How would you like to see it implemented?

1. Enable the Taskchampion support in the app
2. Go to the sidebar
3. Click on CCSync credentials
4. Enter the API url (along with the credentials)

### Would you like to work on this issue?

None

### By submitting this issue, I have confirmed that:

- [X] I have starred the repo ⭐ and watched 👀 it on GitHub and followed the contribution guidelines.

**Comment by c-o-d-e-y-o-d-a** [comment]
Hello, I have experience with Flutter development. Can you please assign this issue to me?

**Comment by its-me-abhishek** [comment]
are there any updates @c-o-d-e-y-o-d-a ?

**Comment by c-o-d-e-y-o-d-a** [comment]
Had exams going on, they end tomorrow. Sorry for the delay, I will open a PR solving the issue as soon as possible

**Comment by its-me-abhishek** [comment]
are there any updates @c-o-d-e-y-o-d-a ?

**Comment by SGI-CAPP-AT2** [comment]
can I work on this one?

**Comment by its-me-abhishek** [comment]
Yeah, i think it should be fine if they are not already working on the issue.

**Comment by SGI-CAPP-AT2** [comment]
Hey, I've created pull request regarding this.
